### PR TITLE
Added the JTransform library (version 2.4.0) for FFT work

### LINKED ISSUE #172: Partially ported model subpackage from neurolab-desktop
**Actual Behaviour**
* The model subpackage(_io.neurolab.model_) is partially ported with five missing classes namely OSCForwardMask, GenericFeedbackSettings, FileOutputTask, FFTPreprocessor and DataPacket.

**Expected Behaviour**
* All the classes should be ported for the completion of the package as it would aid in the implementation of all the four Program Modes and carrying out further tests and validation on this module.

**Classes to Port and Checklist of the work** 
- [x] Adding JTransform or equivalent library.
- [x] DataPacket
- [x] FFTPreprocessor
- [x] OSCForwardMask
- [x] GenericFeedbackSettings
- [x] FileOutputTask

**Would you like to work on the issue?**
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #172 partly (Targeting the second checklist point)

**Changes**: 
* Added JTransforms (version-2.4.0), an open source multithreaded FFT library written in pure Java, for carrying out the FFT related and other math-intensive work.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@iamareebjamal again the same problem (building successfully locally but not in Travis). Will look into it and update the PR shortly. In the meanwhile, if you figure out the reason please do let me know. Thanks.

**Comment by iamareebjamal** [comment]
Exclude junit from the added dependency

**Comment by suresh-gandhi** [comment]
@iamareebjamal did the changes. Let me know if something else needs to be done. Please review.

File: app/build.gradle [hunk 1]
```diff
@@ -46,4 +46,6 @@ dependencies {
     implementation 'com.github.iamareebjamal:jsyn:16.8.0a'
     implementation 'com.illposed.osc:javaosc-core:0.3'
     implementation 'org.scream3r:jssc:2.8.0'
+    def withoutJUnit = {exclude group: 'junit'}
```
Comment (iamareebjamal) [inline app/build.gradle]: No need to assign, just directly use this

**Comment by suresh-gandhi** [comment]
@iamareebjamal did the changes. Let me know if something else needs to be changed. Please review.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**